### ISSUE TITLE
Index new data when anchored on chain

### ISSUE THREAD

**Issue by paul121** [cambiumteam/pinning_service#3]
- Indexer should specify which resolver(s) to subscribe to by resolver ID: https://buf.build/regen/regen-ledger/docs/main:regen.data.v1#DefineResolver
- Data is indexed in named graph using the anchored data IRI
- Default graph includes the IRI and timestamp + txhash

**Comment by paul121** [comment]
Just a note here...

Apparently the websocket RPC event subscription API is deprecated in tendermint v0.36 and will be removed in v0.37:
- https://docs.tendermint.com/master/roadmap/roadmap.html#rpc-event-subscription
- https://github.com/tendermint/tendermint/blob/master/docs/architecture/adr-075-rpc-subscription.md

It seems that regen-ledger 4.0 will be on cosmos-sdk v0.45 https://github.com/regen-network/regen-ledger/blob/7c6081ebff6d358d492480e7564eedf957705a1e/go.mod#L7, which appears to be tendermint 0.34.x.

So tldr; we'll have to move off of websockets eventually but this is all we have available right now

**Comment by paul121** [comment]
Good info about the websock subscribe params: https://pkg.go.dev/github.com/bdware/tendermint/rpc/core#Subscribe
